presto-tests/src/main/java/com/facebook/presto/tests/tpch/AppendingRecordSet.java

```diff
@@ -165,6 +165,12 @@ public Slice getSlice(int field)
         }
 
         @Override
+        public Object getObject(int field)
+        {
+            throw new UnsupportedOperationException();
```

Comment:
But TPCH doesn't have structural types. Implementing it will be adding dead code.